experiments/optimal-core-allocation/README.md

```diff
@@ -37,15 +37,14 @@ where `number of maximum threads dedicated to memcached` is by default set to nu
 sudo optimal-core-allocation -experiment_peak_load=800000
 ```
 
-2. To print results from the experiment, you need to copy `experiment id` in form of a UUID e.g. `ca24aa5d-4a88-7258-6f00-5f651b0d6515`.
-The ID is printed to the standard output and should be used in python functions to generate result tables in Jupyter, like this:
+2. To display results of the experiment, you need to copy `experiment id` (eg `ca24aa5d-4a88-7258-6f00-5f651b0d6515`) and pass it as a parameter to python functions on Jupyter notebook:
```

Comment:
There is no way told how to get Experiment_ID. 
For example, in main readme I have written `sudo memcached-sensitivity-profile (...) > uuid.txt` so that user can get uuid without following logs.